xbmc/guilib/GUIToggleButtonControl.cpp

```diff
@@ -11,6 +11,7 @@
 #include "GUIDialog.h"
 #include "GUIInfoManager.h"
 #include "GUIWindowManager.h"
+#include "guilib/GUIControl.h"
```

Comment:
Is this include really needed?

Reply:
I would say no, as this class derives from CGUIControl which must include GUIControl.h in the header. (https://github.com/xbmc/xbmc/blob/b9e4a3438cfb8b07c9c80d0c5f8c80c4f48dabf7/xbmc/guilib/GUIButtonControl.h#L17)

Reply:
Man, you guys are quick. VS inserted that for me (: